modules/org.jkiss.utils/src/org/jkiss/utils/StringUtils.java

```diff
@@ -89,4 +89,43 @@ public static String truncateText(@NotNull String str, int maxLength) {
         return str;
     }
 
+    /**
+     * Converts an {@code under_score} string to {@code camelCase}.
+     *
+     * {@snippet id="underScoreToCamelCaseExample" lang="java" :
+     * String value = underScoreToCamelCase("some_field");
+     * // value == "someField"
+     * }
+     */
+    @Nullable
+    public static String underScoreToCamelCase(@Nullable String str) {
```

Comment:
Can we please have a test for it? :^)

Reply:
Btw, `underscore` is a single word

```suggestion
    public static String underscoreToCamelCase(@Nullable String str) {
```